README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
 * Maeve Murphy
 * Reagan Smith
 
-## MVP objective
+## Architecture
+
+All data processing is completed using Python 3. Ruby will use the Python scripts to display the resulting graphs on web pages, titled "Overall Distribution" and "Individual Statistics."
+
+### Database Design
+
+The database is implemented using SQLite, and follows this structure:
+
+### Prerequisites
 
-For our minimum viable product (MVP), we expect to deliver two main functionalities: a method to import survey data and a basic graphical representation of the data. To import survey data, we plan on allowing the user to enter a json with multiple people’s results from the survey. This data will then be stored in our database. The second part of our MVP is graphical interpretations of the data. For the MVP, we plan on visualizing the overall results of the data so that the user can see the proportions of challenger to communicator to collaborator to contributor for the class. We also plan to have a set of graphs that show a person’s individual scores and a set of graphs that will show the distribution of responses to each question. 
+This application requires:
+* Python 3.x
+* Rails 5 with Ruby 2.2 or higher
+    - Can be installed by following [these](http://railsapps.github.io/installing-rails.html) instructions
+* Git
+
+### Gems
+
+* High Voltage [found here](https://github.com/thoughtbot/high_voltage)
+
+## Installation
+
+To use this app, you must first download the files. To do so, open up a terminal on your personal machine, navigate to the folder in which you'd like the files to exist, and run these commands
+- `git clone https://github.com/nathanljustin/teamwork-analysis.git`
+- `cd teamworkApp`
+- `rails db:migrate`
+- `rails s`
```

Comment:
This would probably be easier to read and copy if it was one code block:
```
git clone https://github.com/nathanljustin/teamwork-analysis.git
cd teamworkApp
rails db:migrate
rails s
```